.gitignore

```diff
@@ -19,3 +19,5 @@ test/versioned-external/TEMP_TESTS
 nr-security-home
 # Needed for testing
 !test/integration/moduleLoading/node_modules
+# benchmark results
+benchmark_*.json
```

Comment:
Does this mean that the benchmark tests are going to dump results files into the root of the project? If so, can we make them go to a common directory, e.g. `<project_root>/benchmark_results/`?

Reply:
yea, that's prob a good idea